web-components/src/components/floating-modal/FloatingModal.ts

```diff
@@ -50,13 +51,23 @@ export namespace FloatingModal {
         if (this.container && this.show) {
           this.setContainerRect();
           this.setInteractInstance();
+          this.activateFocusTrap!();
+          this.setFocusableElements!();
```

Comment:
Shivam, for correct work focus inside modal, 
  1. move those function `this.activateFocusTrap!(); this.setFocusableElements!();` to the end of setInteractInstance() function, like this:
  `private setInteractInstance() {
      requestAnimationFrame(() => {
        if (this.container) {
          interact(this.container)
            .resizable({
              edges: { left: true, right: true, bottom: true, top: true },
              listeners: {
                end: this.resizeEndListener,
                move: this.resizeMoveListener
              },
              modifiers: this.aspectRatio
                ? [
                    interact.modifiers.aspectRatio({
                      ratio: "preserve",
                      equalDelta: true,
                      modifiers: [interact.modifiers.restrictSize({ max: "parent" })]
                    })
                  ]
                : undefined
            })
            .draggable({
              autoScroll: true,
              allowFrom: this.header,
              ignoreFrom: this.body,
              listeners: {
                move: this.dragMoveListener,
                end: this.dragEndListener
              }
            });
        }
      });
      this.activateFocusTrap!();
      this.setFocusableElements!();
    }`